google-cloud-datastore/src/test/java/com/google/cloud/datastore/StructuredQueryTest.java

```diff
@@ -37,7 +37,9 @@ public class StructuredQueryTest {
   private static final Cursor END_CURSOR = Cursor.copyFrom(new byte[] {10});
   private static final int OFFSET = 42;
   private static final Integer LIMIT = 43;
-  private static final Filter FILTER =
+  private static final Filter AND_FILTER =
+      CompositeFilter.and(PropertyFilter.gt("p1", 10), PropertyFilter.eq("a", "v"));
+  private static final Filter OR_FILTER =
       CompositeFilter.and(PropertyFilter.gt("p1", 10), PropertyFilter.eq("a", "v"));
```

Comment:
Is this a typo where this should be `CompositeFilter.or`?

Reply:
good catch, thanks!